test/stock-count.spec.js

```diff
@@ -1,95 +1,60 @@
-const { fork } = require('child_process');
+const { spawnSync } = require('child_process');
 const path = require('path');
 const fs = require('fs-extra');
-const { once } = require('events');
-const ExcelJS = require('exceljs');
 
 const { stockCountScenario } = require('./mocks/mocks');
 const { 
   setDirToprojectConfig,
-  revertBackToProjectHome
+  revertBackToProjectHome,
+  cleanUp,
+  readDataFromXforms
 } = require('./test-utils');
 
 
 describe('Stock count', () => {
+  const workingDir = process.cwd();
+  const createdAppFormFiles = ['stock_count.properties.json', 'stock_count.xlsx'];
+
 
   beforeEach(() => {
     setDirToprojectConfig();
   });
 
   afterEach(() => {
-    revertBackToProjectHome(process.cwd());
+    cleanUp(workingDir, createdAppFormFiles);
+    revertBackToProjectHome(workingDir);
   });
 
-  it('Add stock count summaries test', async() => {
+  it('Add stock count integration test', async() => {
```

Comment:
The title should be what we expect the test to do. Like: "add stock count feature, stock count file created and list of product added"... I mean, the test performed and the result expected

Reply:
@ernestoteo , this is still unresolved